src/mongo/MongoTripodConfig.class.php

```diff
@@ -688,55 +808,69 @@ public function getCardinality($dbName,$collName,$qName=null)
 
     /**
      * Returns a boolean reflecting whether or not the database and collection are defined in the config
-     * @param string $dbName
-     * @param string $collName
+     * @param string $storeName
+     * @param string $pod
      * @return bool
      */
-    public function isCollectionWithinConfig($dbName,$collName)
+    public function isPodWithinStore($storeName,$pod)
```

Comment:
totally not serious but I think my dyslexia is kicking in I read that method name as 'iPodIsInStore' ... there's some subliminal product placement going on here :p